packages/events/src/index.test.ts

```diff
@@ -0,0 +1,37 @@
+import { vi, test, beforeAll, afterEach, expect } from 'vitest'
```

Comment:
We might want to define these as globals. something like this might be close

```
export default defineConfig({
  test: {
    globals: true,
    }
});